samcli/lib/generated_sample_events/events/lex-v2/LexBookCar.json

```diff
@@ -0,0 +1,98 @@
+{
+    "sessionId": "254688924456220",
+    "inputTranscript": "Book a car",
+    "interpretations": [
+        {
+            "intent": {
+                "slots": {
+                    "ReturnDate": null,
+                    "PickUpDate": null,
+                    "DriverAge": null,
+                    "CarType": null,
+                    "PickUpCity": null
+                },
+                "confirmationState": "None",
+                "name": "BookCar",
+                "state": "InProgress"
+            },
+            "nluConfidence": 1
+        },
+        {
+            "intent": {
+                "slots": {
+                    "RoomType": null,
+                    "CheckInDate": null,
+                    "Nights": null,
+                    "Location": null
+                },
+                "confirmationState": "None",
+                "name": "BookHotel",
+                "state": "InProgress"
+            },
+            "nluConfidence": 0.51
+        },
+        {
+            "intent": {
+                "slots": {},
+                "confirmationState": "None",
+                "name": "FallbackIntent",
+                "state": "InProgress"
+            }
+        }
+    ],
+    "proposedNextState": {
+        "intent": {
+            "slots": {
+                "ReturnDate": null,
+                "PickUpDate": null,
+                "DriverAge": null,
+                "CarType": null,
+                "PickUpCity": null
+            },
+            "confirmationState": "None",
+            "name": "BookCar",
+            "state": "InProgress"
+        },
+        "dialogAction": {
+            "slotToElicit": "PickUpCity",
+            "type": "ElicitSlot"
+        }
+    },
+    "responseContentType": "text/plain; charset=utf-8",
```

Comment:
I believe the `responseContentType` should be in the following format:
```
"responseContentType": "CustomPayload | ImageResponseCard | PlainText | SSML",
```

Reply:
Not for a chat bot when testing